lib/rubocop/cop/rails/action_controller_flash_before_render.rb

```diff
@@ -3,7 +3,7 @@
 module RuboCop
   module Cop
     module Rails
-      # Using `flash` assignment before `render` in Rails controllers will persist the message for too long.
+      # Using `flash` without a following `redirect_to` in Rails controllers will persist the message for too long.
```

Comment:
The original description may be better, as it will match the cop name. So, it does look like the original description is not wrong. Of course, adjustment is welcome. It would be more better if it could be updated while retaining the intent of the original :-)